lib/components/tab.js

```diff
@@ -54,7 +55,7 @@ export default class Tab extends React.PureComponent {
           {this.props.customChildrenBefore}
           <span
             className={`tab_text ${isLast ? 'tab_textLast' : ''} ${isActive ? 'tab_textActive' : ''}`}
-            onClick={this.handleClick}
```

Comment:
Why do you remove this handler (and keep it on `<li>` component)?

If you add onMouseUp, I suppose that is because onClick is not triggered with the middle button?
I would suggest splitting `handleClick` handler (because you have to test `isMiddleClick` only in a `onMouseUp` handler)